src/test/java/api/loans/CheckInByBarcodeTests.java

```diff
@@ -63,6 +65,18 @@
 import io.vertx.core.json.JsonObject;
 
 public class CheckInByBarcodeTests extends APITests {
+
+  @Before
+  public void setupNoteType() {
+    noteTypeFixture.generalNoteType();
```

Comment:
I see there are few places when this initialization is duplicated, could you please move it to the base class - `api.support.APITests#beforeEach`